felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -4061,6 +4145,15 @@ func (m *bpfEndpointManager) getIfaceTypeFromLink(link netlink.Link) IfaceType {
 	return IfaceTypeData
 }
 
+func (m *bpfEndpointManager) getBondSlaves(masterIfName string) []string {
+	slaveDevices := []string{}
+
+	if val, ok := m.hostIfaceToSlaveDevices[masterIfName]; ok {
+		slaveDevices = append(slaveDevices, val.Slice()...)
```

Comment:
you do not need to append to the empty slice above, just return this slice, right? And return nil if there is no set :thinking: